src/cli/commands/floodsub/subscribe.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const utils = require('../../utils')
+const debug = require('debug')
+const log = debug('cli:floodsub')
+log.error = debug('cli:floodsub:error')
+
+module.exports = {
+  command: 'subscribe <topic>',
```

Comment:
Is the intention for this to be aliased from the top level as: `jsipfs sub <some topic>` or from the module as: `jsipfs floodsub sub <topic>`?